pkgs/development/python-modules/decorator/tests.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/src/tests/documentation.py b/src/tests/documentation.py
+index 9dd843a..552b438 100644
+--- a/src/tests/documentation.py
++++ b/src/tests/documentation.py
+@@ -1866,10 +1866,8 @@ def singledispatch_example2():
+ 
+ @decorator
+ def warn_slow(func, duration=0, *args, **kwargs):
+-    t0 = time.time()
+     res = func(*args, **kwargs)
+-    dt = time.time() - t0
+-    if dt >= duration:
++    if duration == 0:
+         print('%s is slow' % func.__name__)
```

Comment:
why the patch? you're not using it in the expression

Reply:
Oh, right, it's an old patch, but then I realized this patch doesn't work because there was already one in the repository.